rolling/distribution.yaml

```diff
@@ -3183,8 +3183,8 @@ repositories:
     release:
       tags:
         release: release/rolling/{package}/{version}
-      url: https://github.com/ros2-gbp/kinematics_interface_pinocchio-release.git
-      version: 0.0.1-1
+      url: git@github.com:ros2-gbp/kinematics_interface_pinocchio-release.git
```

Comment:
This needs to be https, not git.

Reply:
See https://github.com/ros/rosdistro/pull/46690#discussion_r2224155762